src/config/config.ts

```diff
@@ -107,12 +107,14 @@ export class ServerConfigHelpers {
         return (matches) ? matches.map((s:string)=>s.trim()) : [];
     }
 
-    @memoize static priority_studies(str:string|null): PriorityStudies{
+    @memoize static parseConfigFormat(str:string|null): CategorizedConfigItems {
```

Comment:
this seems a very general name for a specific config item format?

Reply:
it's actually is now somewhat generalized, which is why i changed the name.  it used to be just for priority_studies, but now i'm using also for these quick select buttons.   really we need to move this kind of structured configuration to json, which i think we will do soon.  so, hopefully this stuff won't be around much longer.

Reply:
okay, I guess that's fine